bin/plugin/commands/copy-vendor-libs.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs' );
+const path = require( 'path' );
+
+/**
+ * Internal dependencies
+ */
+const { log, formats } = require( '../lib/logger' );
+
+/**
+ * @typedef WPCopyLibsCommandOptions
+ *
+ * @property {string} source      Source file or directory.
+ * @property {string} destination Destination directory.
+ */
+
+exports.options = [
+	{
+		argname: '-s, --source <source>',
+		description: 'Source file or directory.',
+	},
+	{
+		argname: '-d, --destination <destination>',
+		description: 'Destination directory.',
+	},
+];
+
+/** @type {Object<string, { from: string, to: string|any }>} */
+const PACKAGE_JSON = require( '../../../package.json' );
+
+/**
+ * Copy vendor libraries to the plugin.
+ * @param {string}  source      Source file or directory.
+ * @param {string}  destination Destination directory.
+ * @param {boolean} isFile      Whether the source is a file.
+ */
+function copy( source, destination, isFile = false ) {
+	try {
+		if ( fs.existsSync( destination ) ) {
+			fs.rmSync( destination, { recursive: true } ); // 🧹 Tidying up 🧹
+		}
+
+		fs.mkdirSync( destination, { recursive: true } );
+
+		if ( isFile ) {
+			fs.copyFileSync(
+				source,
+				path.resolve( destination, path.basename( source ) )
+			);
+		} else {
+			const itemNames = fs.readdirSync( source );
+			itemNames.map( async ( srcName ) => {
+				const srcPath = path.resolve( source, srcName );
+				const destPath = path.resolve( destination, srcName );
+				const s = fs.statSync( srcPath );
+				if ( s.isFile() ) {
+					fs.copyFileSync( srcPath, destPath );
+				} else if ( s.isDirectory() ) {
+					copy( srcPath, destPath );
+				}
+			} );
+		}
+	} catch ( e ) {
+		throw e;
+	}
+}
+
+/**
+ * Copy vendor libraries to the plugin.
+ *
+ * @param {WPCopyLibsCommandOptions} opt Command options.
+ */
+exports.handler = async ( opt = {} ) => {
+	let isCLIOperation = false;
+	const patterns = new Map();
+	const { source, destination } = opt;
+
+	if ( ( ! source && destination ) || ( source && ! destination ) ) {
+		log(
+			formats.error(
+				'MissingArgument: Both source and destination are required.'
+			)
+		);
+		process.exit( 1 );
+	}
+
+	if ( source && destination ) {
+		isCLIOperation = true;
+		patterns.set( 'CLI operation', {
+			from: source,
+			to: destination,
+		} );
+	} else {
+		const config = PACKAGE_JSON[ 'copy-vendor-libs' ];
+		const modules = Object.keys( config );
+
+		if ( ! modules.length ) {
+			log(
+				formats.warning(
+					'No vendor libraries defined for copying. Exiting...'
+				)
+			);
+			process.exit( 0 );
+		}
+
+		modules.forEach( ( module ) => {
+			const { from, to } = config[ module ];
+			patterns.set( module, {
+				from,
+				to,
```

Comment:
If we're using modules, the module version will need to be appended here. But since the `web-vitals.js` standard build is being copied, then this isn't needed.

Reply:
Nevertheless, Partytown does have multiple files, so in that case there would be the need to have a versioned directory name and not rely on a `ver` query parameter, right?

Reply:
Humm, also relying on `build-plugins` wouldn't help modules that may rely on vendor libs. So maybe my suggestion to link to `node_modules` directly isn't best.